hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,114 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "vlib work && {job_prefix} vlog"
+  run_cmd:    "{job_prefix} vsim"
+
+  // Indicate the tool specific helper sources - these are copied over to the
+  // {tool_srcs_dir} before running the simulation.
+
+  build_opts: ["-sv",
+               "-timescale 1ns/1ps",
+               "+incdir+/${QUESTA_HOME}/verilog_src/uvm-1.2/src/",
+               "/${QUESTA_HOME}/verilog_src/uvm-1.2/src/uvm_pkg.sv",
```

Comment:
DVSim treats items in braces as variables. The preceding `/$` should not be needed. Please give below a try and let me know if that is not working. 

```suggestion
               "+incdir+{QUESTA_HOME}/verilog_src/uvm-1.2/src/",
               "{QUESTA_HOME}/verilog_src/uvm-1.2/src/uvm_pkg.sv",
```